src/authentication/Revoked.jsx

```diff
@@ -0,0 +1,45 @@
+/* global cozy */
```

Comment:
Replaces the former `RevokableWrapper`. Note that it doesn't render the routes as childrens anymore as it xas a bit buggy.